sql/src/main/java/io/crate/analyze/symbol/Symbols.java

```diff
@@ -149,6 +149,11 @@ public Boolean visitFetchReference(FetchReference fetchReference, ColumnIdent co
         }
 
         @Override
+        public Boolean visitField(Field field, ColumnIdent needle) {
```

Comment:
why is the context called `needle` here?

Reply:
Because it's the column that is being looked for. `searchTerm` doesn't really fit, so I choose needle because that's quite common in some APIs  (haystack/needle)